libsql_graph_db/natural.py

```diff
@@ -1,3 +1,35 @@
-"""
-Use models.py and Instructor to provide a natural language interface to the graph db
-"""
+import os
+import instructor
+from openai import OpenAI
+from dotenv import load_dotenv
+from models import KnowledgeGraph  # type: ignore
+from database import initialize, add_node, atomic  # type: ignore
```

Comment:
`# type: ignore` this should not be required for files written by us, if you are needing this - that means some types are probably missing in that file. Look at the mypy errors and fix the typing issues there

Reply:
Done